src/hooks/useChainId.ts

```diff
@@ -41,7 +42,7 @@ export const useUrlChainId = (): string | undefined => {
   if (shortName) {
     const chainId = Object.entries(chains).find(([key]) => key === shortName)?.[1]
     if (chainId == null) {
-      throw Error('Invalid chain short name in the URL')
+      router.push(AppRoutes.welcome)
```

Comment:
This hook is called from 150+ components. It's supposed to be a getter-only hook.

I suggest we handle bogus short names outside of this hook. E.g. somewhere in _app.tsx. This hook could be wrapped in a try-catch there.

Reply:
OK.